sdkit/models/model_loader/vae.py

```diff
@@ -1,49 +1,80 @@
+"""
+    VAE model loader
+"""
 import os
 import traceback
 import tempfile
 
 from sdkit import Context
 from sdkit.utils import log, load_tensor_file
 
+
 """
 The VAE model overwrites the state_dict of model.first_stage_model.
 
 We keep a copy of the original first-stage state_dict when a SD model is loaded,
 and restore that copy if the custom VAE is unloaded.
 """
 
+
 def load_model(context: Context, **kwargs):
+    """
+        Load the model.
+    """
     vae_model_path = context.model_paths.get('vae')
 
     try:
         vae = load_tensor_file(vae_model_path)
-        vae_dict = {k: v for k, v in vae["state_dict"].items() if k[0:4] != "loss"}
+        vae_dict = {
+            k: v
+            for k, v in vae["state_dict"].items()
+            if k[0:4] != "loss"
+        }
         if context.half_precision:
             for key in vae_dict.keys():
                 vae_dict[key] = vae_dict[key].half()
 
         _set_vae(context, vae_dict)
 
         del vae_dict
-        return {} # we don't need to access this again
+        # we don't need to access this again
+        return {}
     except:
-        log.error(traceback.format_exc())
-        log.error(f'Could not load VAE: {vae_model_path}')
+        log.error('%s', traceback.format_exc())
+        log.error('Could not load VAE: %s', vae_model_path)
+    return None
+
 
-def move_model_to_cpu(context: Context):
-    pass
+def move_model_to_cpu(_: Context):
+    """
+        Move the model to CPU.
 
-def unload_model(context: Context, **kwargs):
+        # TODO: Implement this?
+    """
+
+
+def unload_model(context: Context, **__):
```

Comment:
Redundant documentation, and in internal modules where the documentation isn't necessary. This bloats up the lines of code unnecessarily.

Reply:
Does documentation on method has an impact on how the code run? As it stand it isn't very useful but maybe that need to change to be more descriptive. It will help anyone looking at the code for the first time.